cotainr/pack.py

```diff
@@ -261,8 +262,10 @@ def _display_miniforge_license_for_acceptance(self, *, installer_path):
                 "\n",
             )
             logger.debug(f"The Miniforge displayed license is: {license_text}")
-            val = input(license_text)  # prompt user for acceptance of license terms
-            if val != "yes":
+            answer = util.answer_yes(
+                license_text
+            )  # prompt user for acceptance of license terms
+            if not answer:
```

Comment:
Weird formatting for the comment, and I don't think it really adds much, so...
```suggestion
            if not util.answer_yes(license_text):
```

Reply:
Though it doesn't seem like a big thing i reckon that the license acceptance is a pivotal point in the program and i'd like to keep the comment as is.